docs/user/rstudio/ide/guide/code/diagnostics.qmd

```diff
@@ -2,26 +2,23 @@
 title: "Code Diagnostics"
 ---
 
-Copied from <https://support.rstudio.com/hc/en-us/articles/205753617-Code-Diagnostics-in-the-RStudio-IDE>
-
 ## Enabling Diagnostics
 
```

Comment:
It may be helpful to include a general sentence here explaining what code diagnostics is/does. I think the word "Diagnostics" may not mean much to the average R user so some info right at the beginning about how the IDE can automatically check and alert you about certain errors in your code will be attention grabbing.

Reply:
added:

```
RStudio has the ability to perform static and dynamic analysis of R code or what is known as "Diagnostics". 
Diagnostics are a way to indicate stylistic issues or errors with the code.
```

Reply:
Beautiful!